use-timescale/metrics-logging/insights.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Insights
+excerpt: Get query-level performance insights, such as slow queries, memory and data access patterns, and execution metrics
+products: [cloud]
+keywords: [metrics, monitoring, services, query, performance, insights]
+tags: [dashboard, cpu, memory, storage, query, performance]
+cloud_ui:
+    path:
+        - [services, :serviceId, insights]
+---
+
+# Insights
+
+Insights allows you to gain a comprehensive understanding of how your queries perform over time. It empowers you to optimize your queries and make the most efficient use of your instance resources.
+
+You can view query-level insights from the `Insights` tab at your service Cloud Dashboard.
```

Comment:
Should we say this tool provides insights on queries performed in the last 24 hours?

Reply:
I think we should, good idea! At least clarify the maximum date range we currently support!

Reply:
@vineethapai, what do you think to rephrase to: You can view up to 24 hours of query-level insights from the `Insights` tab on your service Cloud Dashboard.

Reply:
Can we say this "On the insights tab, you can access insights on queries and instance resources for the previous 24 hours."

Reply:
For sure! I'll change it right now

Reply:
Done ✅ Waiting the preview just to confirm, but the change is already done